# Request
---
bool success
string message
float64[] joints
